azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -66,22 +59,11 @@
  */
 @Singleton
 @Deprecated
-public class ExecutorManager extends EventHandler implements
-    ExecutorManagerAdapter {
+public class ExecutorManager extends AbstractExecutorManagerAdapter {
```

Comment:
If the PUSH dispatch model will still be supported maybe we should remove the @Deprecated annotation on top.

Reply:
It is good to keep it deprecated as it has many limitations. Preferable solution for now is POLL and in future CONTAINERIZED will be added.